Remotion/Web/Core/ExecutionEngine/UrlMapping/UrlMappingConfiguration.cs

```diff
@@ -55,11 +58,11 @@ public static UrlMappingConfiguration Current
 
   private static UrlMappingConfiguration CreateConfig ()
   {
-    var mappingFile = SafeServiceLocator.Current.GetInstance<WxeUrlSettings>().UrlMappingFile;
-    if (string.IsNullOrEmpty(mappingFile))
-      return new UrlMappingConfiguration();
-    else
-      return CreateUrlMappingConfiguration(mappingFile);
+    var urlMappingLoader = new UrlMappingLoader(
```

Comment:
why do we not get UrlMappingLoader from IoC? Please add registration for UrlMappingLoader and simplfy.